src/vk/VkClearColorValue.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Vulkan
+{
+    /// <summary>
+    /// Structure specifying a clear color value.
+    /// </summary>
+    /// <summary>
+    /// Structure specifying a clear color value.
+    /// </summary>
+    [StructLayout(LayoutKind.Explicit)]
+    public struct VkClearColorValue
+    {
+        [FieldOffset(0)] public VkColor4 Float4;
+        [FieldOffset(0)] public VkColorI4 Int4;
+        [FieldOffset(0)] public VkColorU4 UInt4;
+
+        public VkClearColorValue(float r, float g, float b, float a = 1.0f) : this()
```

Comment:
I think this constructor is a good addition, but I don't really like the new structure types used below. I think if we just had 3 constructors, one for each primitive type, it would cover 99% of use cases without having a new custom structure. Realistically speaking, I don't think anybody is going to be storing their data in those custom structs, so they would have to convert from their representation into a VkClearColorValue anyways. So having to call this overload (with 4 individual parameters) is about the same, IMO.